docs/declarative-customization/column-formatting.md

```diff
@@ -880,24 +973,13 @@ You can use predefined icons from Office UI Fabric. For details, see the [Fabric
 
 ## Creating custom JSON
 
-Creating custom column formatting JSON from scratch is simple if you understand the schema. To create your own custom column formatting:
-
-1. [Download Visual Studio Code](https://code.visualstudio.com/Download). It's free and fast to download. 
-
-2. In Visual Studio Code, create a new file, and save the empty file with a .json file extension.
+Creating custom column formatting JSON from scratch is simple if you understand the schema. The Monaco Editor is integrated in the formatting pane with pre-filled JSON column schema reference to assist you to create your own column formatting. You have validation and autocomplete to create your JSON. You can start adding your JSON after the first line that defines the schema location.
```

Comment:
Creating custom column formatting JSON from scratch is simple if user understands the schema, Monaco Editor is integrated in the formatting pane with pre-filled JSON column schema reference to assist in creation of column formatting, Monaco editor has validation and autocomplete to help in crafting right JSON. User can start adding JSON after the first line that defines the schema location.

Reply:
Also fixed corresponding View Formatting doc